ydb/core/tx/schemeshard/ut_helpers/helpers.h

```diff
@@ -361,6 +365,7 @@ namespace NSchemeShardUT_Private {
         NKikimrSchemeOp::EIndexType IndexType = NKikimrSchemeOp::EIndexTypeGlobal;
         TVector<TString> IndexColumns;
         TVector<TString> DataColumns;
+        TVector<NYdb::NTable::TGlobalIndexSettings> GlobalIndexSettings = {};
```

Comment:
Please, evaluate the following design choice.

I have decided to use a structure defined in the public API to describe a TBuildIndexConfig in SchemeShard's unit tests.

Pros:
- We don't have to repeat the same code in the unit test helpers.

Cons:
- Using public API for unit test helpers might limit us greatly. We would have to change the public description of an index every time we decide to test some new functionality in the SchemeShard tests.
  - Counterargument: ut helpers already [use](https://github.com/ydb-platform/ydb/blob/33cb4333c3daf2ab4a78a3964e595b7e24623d5b/ydb/core/tx/schemeshard/ut_helpers/helpers.cpp#L1677) a protobuf from the public API to describe an index to be created. It is only natural to use the SDK to work conveniently with the public protobuf. The dependency on the public API and the limitations that come with it would not be lifted, if we didn't use SDK here.